e2e/jsonschema/match.go

```diff
@@ -66,7 +66,12 @@ func MatchEntities(data []*sdk.EntityData, schemaFileByJobByType map[string]Even
 	}
 
 	for _, entityData := range data {
-		metric := mergeMetricSets(entityData.Metrics)
+		// Merge metricSets from the entity
```

Comment:
Not really needed comment. We could say that we need to merge to be able to validate metrics coming from different sources using single JSON scheme.